src/redux/actions/notifications.action.js

```diff
@@ -61,3 +61,45 @@ export const clearNotify = () => async (dispatch) => {
     dispatch(clearFailure(error));
   }
 };
+
+const optInOutSuccess = (data) => ({
+  type: OPTINOUT_SUCCESS,
+  payload: data
+});
+
+const optInOutError = (error) => ({
+  type: OPTINOUT_ERRORS,
+  payload: error
+});
+
+export const NotifStatusAction = () => async (dispatch) => {
+  try {
+    const token = window.localStorage.getItem('token');
+    const axiosHeader = {
+      headers: {
+        'content-type': 'application/json',
+        Authorization: `Bearer ${token}`
+      },
+    };
+    const response = await fetchNotifStatus(axiosHeader);
+    dispatch(optInOutSuccess(response.data));
+  } catch(error) {
+    dispatch(optInOutError('Notififications cannot be found'));
+  }
```

Comment:
You should also handle network errors, eg if there is no internet connection, the app should not crush.